### PR TITLE
[doc] update terminate topic docs, since terminate is not support for non-p…

### PR DESCRIPTION
**Pull request by gaozhangmin**
Update docs about terminate topic, Because terminate a non-persistent topic is not support.

### CODE REVIEW

**Comment by gaozhangmin** [comment]
@Anonymitaet  PTAL

File: site2/website/static/swagger/2.3.2/swagger.json [hunk 1]
```diff
@@ -4422,54 +4422,6 @@
         }
       }
     },
-    "/non-persistent/{tenant}/{namespace}/{topic}/terminate" : {
```
Comment (Anonymitaet) [inline site2/website/static/swagger/2.3.2/swagger.json:4425]: @tuteng is it correct to delete this content in `swagger.json`?

**Review by Anonymitaet — COMMENTED** [review]
@tuteng is it correct to delete the content in `swagger.json`?

**Comment by gaozhangmin** [comment]
> @tuteng is it correct to delete the content in `swagger.json`?

no terminate api found for non-persistent topic.

**Review by tuteng — APPROVED** [review]

**Pull request merged**